packages/opencensus-core/src/common/validations.ts

```diff
@@ -43,3 +45,27 @@ export function validateArrayElementsNotNull<T>(
     throw new Error(`${errorMessage} elements should not be a NULL`);
   }
 }
+
+/** Throws an error if any of the map elements is null. */
+export function validateMapElementNotNull<T>(
+    map: Map<T, T>, errorMessage: string) {
+  for (const [key, value] of map.entries()) {
+    if (key === null || key === undefined || value === null ||
+        value === undefined) {
+      throw new Error(`${errorMessage} elements should not be a NULL`);
+    }
+  }
+}
+
+/** Throws an error if any of the array element present in the map. */
+export function validateDuplicateKeys(
+    keys: LabelKey[], map: Map<LabelKey, LabelValue>) {
+  const keyNames: Set<string> = new Set();
+  keys.forEach(key => keyNames.add(key.key));
```

Comment:
nit: consider adding a separate check on duplicated keys in `keys`:
```
if (keyNames.size !== keys.length) {
  throw new Error(...);
}
```

Reply:
For simplicity, I have combined with existing check.